devel_requirements.txt

```diff
@@ -1,5 +1,6 @@
 black
 nypl-py-utils[avro-client,config-helper]==1.2.0
 pybase64
-python-csv
-python-io
\ No newline at end of file
+python-csv==0.0.10
+python-io
+pyyaml
```

Comment:
This will be automatically installed by `nypl-py-utils[config-helper]`. Same with `avro` and `nypl-py-utils[avro-client` below. In general, the names between the brackets are just aliases for lists of packages -- if you look at https://github.com/NYPL/python-utils/blob/main/pyproject.toml you can see the packages each name installs.

Reply:
Ahhh gotcha! Making that change now